packages/@okta/i18n/src/properties/login.properties

```diff
@@ -43,6 +43,7 @@ error.unsupported.cors = Unsupported browser - missing CORS support
 error.unsupported.localStorage = Unsupported browser - missing localStorage support
 error.enabled.cors = There was an error sending the request - have you enabled CORS?
 error.expired.session = Your session has expired. Please try to log in again.
+error.mfa.only.expired.session = Unable to authenticate at this time. Try again.
```

Comment:
Please run this by UX to see if they can come up with something better. 
'Try again.'   how ?  there is no button not text on what are the next steps.

Reply:
How is the hard part. In RDP we close the window, in end user settings, we need to go back. There is no sure way of telling a user how to try again. Maybe we should just say `Unable to authenticate at this time.`? @mauriciocastillosilva-okta @vincentvoong-okta thoughts?

Reply:
yeah, it is hard to come up with something accurate that can help, since it really is a dead end error.

We will show this because of an InvalidToken error, which just literally means that: the token is no longer valid. In the regular sign in flow, the user can go back and start over, so we can show a next step to try to start login flow again. However with `mfaOnlyFlow` the user needs a new token to start over, and we don't know **how** they can get that token. Leaving RDP aside, since for SIW purposes we shouldn't think of a specific usage of the widget, I think the error should just tell the user what happened. Maybe just `Your token is invalid.`, or if we follow the same pattern as the regular flows, this will be something like `Unable to validate your credentials at this time.`. If we try to be clever about next steps, then I think what Gowthami has is the best guess we can have, since we don't exactly know where they come from. 

With that said, I think @vincentvoong-okta should make the call